README.md

```diff
@@ -8,56 +8,108 @@
-
-## Installation
-Information about how to install the UL IoTAgent can be found at the corresponding section of the [Installation & Administration Guide](docs/installationguide.md).
+## Testing
+[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Should.js](https://shouldjs.github.io/) Assertion Library.
 
-## API Overview
+The test environment is preconfigured to run BDD testing style.
 
-An Overview of the API can be found in the [User & Programmers Manual](docs/usermanual.md).
+To run tests, type
 
-## API Reference Documentation
+```bash
+npm test
+```
 
-Apiary reference for the Configuration API can be found [here](http://docs.telefonicaiotiotagents.apiary.io/#reference/configuration-api).
-More information about IoTAgents and their APIs can be found in the IoTAgent Library [here](https://github.com/telefonicaid/iotagent-node-lib).
+## Quality Assurance
 
-## Development documentation
+This project is part of [FIWARE](https://fiware.org/) and has been rated as
+follows:
 
-Information about developing for the UL IoTAgent can be found at the corresponding section of the [User & Programmers Manual](docs/usermanual.md).
 
-## Testing
-[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Should.js](https://shouldjs.github.io/) Assertion Library.
+-   **Version Tested:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Version&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.version&colorB=blue)
+-   **Documentation:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Completeness&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.docCompleteness&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Usability&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.docSoundness&colorB=blue)
+-   **Responsiveness:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Time%20to%20Respond&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.timeToCharge&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Time%20to%20Fix&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.timeToFix&colorB=blue)
+-   **FIWARE Testing:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Tests%20Passed&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.failureRate&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Scalability&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.scalability&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Performance&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.performance&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Stability&url=https://fiware.github.io/catalogue/json/iotagent_ul.json&query=$.stability&colorB=blue)
 
```

Comment:
This section should include:

* Information about the last published version (so the user can know the gap between "verstion tested" and the newest one, evaluating if the information is obsolete enough from him or not)
* A link to the labeling calculation procedure, i.e. which metrics and in which form decide which label to assign. Ideally, this should be a mathematical formula/table leaving no room for subjective interpretation.
* A link (or several) to the report(s) in which the labels are based, explaining which particular values of the metrics has been used (and how they has been got) in the above formula.

In addition, why two of them are in "no results" status?

Reply:
All these are good suggestions, but I am unable to implement the requested changes at the moment.

The badges are dynamically driven from the data held in a series of JSON files. The data you are requesting is currently not present within FIWARE. The existing files only hold the **version tested** and not the **current version.** , FIWARE QA don't hold the current version for this component -that is under Telefonica's control

Regarding the testing procedure, the following links within the FIWARE catalogue describe the process:

* https://github.com/Fiware/catalogue#testing
* https://www.fiware.org/developers/qa-labeling-of-fiware-components/

It is not part of the current [contribution requirements](https://github.com/Fiware/contribution-requirements) to display additional links and information - I agree that the QA section could be expanded sometime, but the limited information within this block is currently just a summary of the parts which are available. If you want to display more that is fine, but it would make sense to add a PR to the requirements so that all enablers display the same information consistently.

The two missing ("no results" status) badges are because the QA data has not been updated yet.

Reply:
All these are good suggestions, but I am unable to implement the requested changes at the moment.

The badges are dynamically driven from the data held in a series of JSON files. The data you are requesting is currently not present within FIWARE. The existing files only hold the **version tested** and not the **current version.** , FIWARE QA don't hold the current version for this component -that is under Telefonica's control

Regarding the testing procedure, the following links within the FIWARE catalogue describe the process:

* https://github.com/Fiware/catalogue#testing
* https://www.fiware.org/developers/qa-labeling-of-fiware-components/

It is not part of the current [contribution requirements](https://github.com/Fiware/contribution-requirements) to display additional links and information - I agree that the QA section could be expanded sometime, but the limited information within this block is currently just a summary of the parts which are available. If you want to display more that is fine, but it would make sense to add a PR to the requirements so that all enablers display the same information consistently.

The two missing ("no results" status) badges are because the QA data has not been updated yet.

Reply:
To avoid duplication :) please see https://github.com/telefonicaid/iotagent-json/pull/341/files#r240975502